src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -3572,13 +3572,16 @@
     /**
      * The {@code record} keyword.  This element appears
      * as part of a record declaration.
+     *
+     * @since 8.35
```

Comment:
8.35 was just released so this needs to be upped to 36

Reply:
This was added in 8.35 with the preliminary record support.  Should I make this a separate PR or commit?

Reply:
Sorry, didn't make the connection.